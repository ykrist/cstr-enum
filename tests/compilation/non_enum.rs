use cstr_enum::*;

#[derive(FromCStr, AsCStr)]
struct Struct {
  field: u8
}

fn main() {

}