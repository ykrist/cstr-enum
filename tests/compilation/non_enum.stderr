error: target must be an enum
 --> $DIR/non_enum.rs:3:20
  |
3 | #[derive(FromCStr, AsCStr)]
  |                    ^^^^^^
  |
  = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: target must be an enum
 --> $DIR/non_enum.rs:3:10
  |
3 | #[derive(FromCStr, AsCStr)]
  |          ^^^^^^^^
  |
  = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)
